### PR TITLE
Bump UAA and mysql releases and fixes for openSUSE

### PR DESCRIPTION
**Pull request by viovanov**
This also changes the location of build artifacts, so the UAA images can be built easier while inside a suse/scf vagrant box

### CODE REVIEW

File: .envrc [hunk 1]
```diff
@@ -25,11 +25,12 @@ export FISSILE_RELEASE="${FISSILE_RELEASE#,}"
 export FISSILE_ROLE_MANIFEST="${PWD}/role-manifest.yml"
 export FISSILE_LIGHT_OPINIONS="${PWD}/opinions.yml"
 export FISSILE_DARK_OPINIONS="${PWD}/dark-opinions.yml"
-export FISSILE_WORK_DIR="${PWD}/work-dir"
+export FISSILE_WORK_DIR="$HOME/.fissile/uaa-work-dir"
```
Comment (mook-as) [inline .envrc]: That wouldn't exist if you're developing this locally (outside of any vagrant box type things).
Can we make this fallback instead?
```bash
export FISSILE_WORK_DIR="${FISSILE_WORK_DIR:-$HOME/.fissile/uaa-work-dir}"
```
Then we can set a work dir outside, or something.
Otherwise we can make the `.envrc` smarter and check if this looks like a vagrant box and whatnot, but that might be too much.

Actually, why was this changed in the first place?

File: .envrc [hunk 2]
```diff
@@ -25,11 +25,12 @@ export FISSILE_RELEASE="${FISSILE_RELEASE#,}"
...
 export FISSILE_REPOSITORY="uaa"
+export FISSILE_CACHE_DIR="$HOME/.bosh/cache"
```
Comment (mook-as) [inline .envrc]: Isn't that the default value?

File: opinions.yml [hunk 3]
```diff
@@ -2,6 +2,7 @@
 properties:
   cf_mysql:
     mysql:
+      startup_timeout: 300
```
Comment (mook-as) [inline opinions.yml]: Didn't we just take this out or something?

File: scripts/authorize_internal_ca.sh [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    fi
+fi
+
+if [ -r /etc/secrets/internal-ca-cert ]; then
+    cp /etc/secrets/internal-ca-cert "${ca_path}"/internalCA.crt
+elif [ -n "${INTERNAL_CA_CERT:-}" ]; then
+    printf "%b" "${INTERNAL_CA_CERT}" > "${ca_path}"/internalCA.crt
+fi
+
+if [ -n "${HCP_CA_CERT_FILE:-}" -a -r "${HCP_CA_CERT_FILE:-}" ]; then
```
Comment (mook-as) [inline scripts/authorize_internal_ca.sh]: The `HCP_CA_CERT` section should be removed, there's no occasion here where it would make sense.

File: opinions.yml (hunk 3 above)
Comment (viovanov) [inline opinions.yml]: We "took it out" because it changed name and fissile complained that it couldn't find it.
I don't think the change to _remove_ was deliberate.

File: .envrc (hunk 2 above)
Comment (viovanov) [inline .envrc]: I think it got the value from `scf`, because uaa is nested within in my setup (like it will be as a submodule)

File: .envrc (hunk 1 above)
Comment (viovanov) [inline .envrc]: some permission errors in my setup, i'll take another look

File: .envrc (hunk 2 above)
Comment (mook-as) [inline .envrc]: Oh, right, when it becomes a submodule lots of this can go away (because `source_up`).

File: scripts/authorize_internal_ca.sh (hunk 4 above)
Comment (viovanov) [inline scripts/authorize_internal_ca.sh]: this is 1:1 with scf atm
i'm thinking we keep it like this, and the card to remove hcp/hcf references takes care of it

File: .envrc (hunk 1 above)
Comment (mook-as) [inline .envrc]: We'll clean this up in a separate PR.
